barretenberg/cpp/src/barretenberg/ecc/fields/field_conversion.cpp

```diff
@@ -113,4 +113,19 @@ std::vector<bb::fr> convert_to_bn254_frs(const curve::Grumpkin::AffineElement& v
     return fr_vec;
 }
 
-} // namespace bb::field_conversion
\ No newline at end of file
+grumpkin::fr convert_challenge(const bb::fr& f, grumpkin::fr* /*unused*/)
+{
+    const uint64_t NUM_CONVERSION_TWO_LIMB_BITS = 2 * NUM_CONVERSION_LIMB_BITS;
```

Comment:
I can't change the name since it should be the same function in all cases. Alternative is to create a constructor that does this and have this call that constructor